ceph.spec.in

```diff
@@ -908,6 +908,17 @@ depending on python-ceph should be fixed to depend on python-rados,
 python-rbd, python-rgw or python-cephfs instead.
 %endif
 
+%package grafana-dashboards
+Summary:	The set of grafana dashboards for monitoring purposes
+%if 0%{?suse_version}
+Group:		System/Filesystems
+%endif
+%description grafana-dashboards
```

Comment:
i wonder if grafana is a dependency of `ceph-grafana-dashboards`, shall we list it as a `Requires:` ?

Reply:
This is a matter of opinion. The dashboards will sit there just fine without Grafana, but are useless. Some distros don't like packages like this fr this very reason.

I think its fine to leave as is, but have no strong feeling either way.

Reply:
i see... i thought grafana was packaged for well known distros. but turns out it's not =(

do you think is an option to  "Suggests: pcp-webapp-grafana" or "Suggest: grafana"? so dependency solver will ignore it and show them as hints?

Reply:
Yes that certainly makes sense.

Reply:
I was thinking about requiring grafana, here. The thing is that grafana is likely to be run as a container (with /etc/grafana bind mounted), it wouldn't make much sense for the package to require it in that case.

I was thinking about doing something like `Suggests` but afaik, support for that is not very wide-spread, yet. :-/